docs/JLBP-0006.md

```diff
@@ -82,9 +82,9 @@ Given this scenario, here are the possible combinations of renamings:
     will only choose one of them to use. Users are forced to change their code
     between versions, but only for library surface that was changed.
 
-  - **Case 2: Rename Maven ID**. Users can easily pull in both the old jar
-    (`g1:a1:1.0.0`) and the new jar (`g2:a2:2.0.0`) into their classpath
-    accidentally through transitive dependencies because Maven artifact
+  - **Case 2: Rename Maven ID**. Users can accidentally pull in the JAR files of both the
```

Comment:
Users --> Build system?
Also, I'm not sure it's accidental.

Reply:
Updated to "build systems".

Updated to "unexpectedly".